instrumentation-api-incubator/src/main/java/io/opentelemetry/instrumentation/api/incubator/semconv/db/SqlClientAttributesExtractorBuilder.java

```diff
@@ -48,12 +49,29 @@ public SqlClientAttributesExtractorBuilder<REQUEST, RESPONSE> setStatementSaniti
     return this;
   }
 
+  /**
+   * Sets whether the query parameters should be captured as span attributes named {@code
+   * db.query.parameter.<key>}. Enabling this option disables the statement sanitization. Disabled
+   * by default.
+   *
+   * <p>WARNING: captured query parameters may contain sensitive information such as passwords,
+   * personally identifiable information or protected health info. Exposing such info may result in
+   * substantial fines and penalties or criminal liability. Consult your peers, superiors and a
+   * legal counsel before enabling this option.
```

Comment:
(and elsewhere, based on https://github.com/open-telemetry/community/issues/2713#issuecomment-2877890644)
```suggestion
   * personally identifiable information or protected health info.
```